### PR TITLE
Fixes operator reconcillation on older secret deletion and CreateVolume log fixes.

### PR DESCRIPTION
**Pull request by badri-pathak**
## Pull request checklist

```release-notes

```
PR to address issues:
[Driver Pods are Restarting if Previous Secret is Deleted](https://github.com/IBM/ibm-spectrum-scale-csi/issues/856)
[Fix Fileset creation logs for createVolume](https://github.com/IBM/ibm-spectrum-scale-csi/issues/1179)

## Pull request type

 

 

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature Enhancement
- [ ] Test Automation 
- [ ] Code Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Community Operator listing 
- [ ] Other (please describe): 

## What is the current behavior?

-

## What is the new behavior?

-

## How risky is this change?
- [ ] Small, isolated change
- [x] Medium, requires regression testing
- [ ] Large, requires functional and regression testing

### CODE REVIEW

**Comment by badri-pathak** [comment]
Images: 
quay.io/badri_pathak/ibm-spectrum-scale-csi-operator:secretdel_v2
quay.io/badri_pathak/ibm-spectrum-scale-csi-driver:filsetexistscheck_v2

**Comment by saurabhwani5** [comment]
Hi @badri-pathak , can you help in resolving conflicts

**Comment by badri-pathak** [comment]
@saurabhwani5 addressed the conflict issue.

**Review by saurabhwani5 — DISMISSED** [review]
working as expected

File: driver/csiplugin/connectors/rest_v2.go [hunk 1]
```diff
@@ -848,6 +848,10 @@ func (s *SpectrumRestV2) ListFileset(ctx context.Context, filesystemName string,
 
 	err := s.doHTTP(ctx, getFilesetURL, "GET", &getFilesetResponse, nil)
 	if err != nil {
+		if strings.Contains(err.Error(), "Invalid value in 'filesetName'") { // This means fileset is not present, create it
+			klog.V(6).Infof("[%s] Fileset already  with name %s doesn't exists.", utils.GetLoggerId(ctx), filesetName)
```
Comment (hemalathagajendran) [inline driver/csiplugin/connectors/rest_v2.go:852]: Rephrase the sentence.

Comment (badri-pathak) [reply driver/csiplugin/connectors/rest_v2.go]: addressed the comment

**Review by hemalathagajendran — APPROVED** [review]

**Pull request merged**